src/FFDHE.h

```diff
@@ -0,0 +1,22 @@
+// Copyright 2005-2017 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_FFDHE_H_
+#define MUMBLE_FFDHE_H_
+
+/// FFDHE provides access to the Diffie-Hellman parameters from RFC 7919.
+class FFDHE {
+	public:
+		/// PEMForNamedGroup returns the PEM-encoded
+		/// Diffie-Hellman parameters for the RFC 7919
+		/// group with the given name, such as "ffdhe2048",
+		/// "ffdhe4096", etc.
+		///
+		/// Returns an empty and null byte array if the request
```

Comment:
Is null even a thing for QByteArray? You are referring to its data struct behind the visible QByteArray being null, right? Like on QString? But QString explicitly talks about that

> Constructs a null string. Null strings are also empty.

while QByteArray does not expose that information.